pom.xml

```diff
@@ -56,6 +56,12 @@
             <version>5.8.2</version>
             <scope>test</scope>
         </dependency>
+        
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.12.0</version>
+        </dependency>
```

Comment:
I am generally against adding dependencies that bloat the final program for tasks that can be implemented by hand with minor effort, as this tends to end with maintenance headaches.

In particular, in this case, one could resort to `System.getProperty("os.name").toLowerCase()` and use `.contains()` to check the OS. This could be mapped to an `enum` to avoid passing around strings and enclosed in a dedicated class.

Reply:
It is! But (maven) dependency management is painful, so I prefer to rely on the smallest number of dependencies in general.